pkg/components/linkerd/manifest.go

```diff
@@ -19,6 +19,15 @@ const chartValuesTmpl = `
 global:
   identityTrustAnchorsPEM: |
 {{ .CA }}
+  proxy:
```

Comment:
This is not related to HA settings, but it's the resources limits.

Reply:
It is directly copied from HA values file.